source/administration/feature-documentation-template.rst

```diff
@@ -0,0 +1,37 @@
+Feature Documentation Template
+===============================
+
+
+This template defines how a feature should be documented improving presentation
+and retrieval of information of product features. The document is user focused.
+
+Enabling The Feature
```

Comment:
We should also include any prerequisites as well.  Do you think that would be it's own section or part of the "Enabling the feature" section?